legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/result/ResultNormalizer.java

```diff
@@ -71,7 +71,7 @@ public static Object normalizeToSql(Object o, String databaseTimeZone)
 
         if (o instanceof String)
         {
-            return ((String) o).replace("'", "\'").replace("\\", "\\\\");
+            return ((String) o).replace("\\", "\\\\");
```

Comment:
Does the test TestResultNormalizer.testStringNormalization still pass with this change? Is the replace that is removed happening somewhere else now?

Reply:
replace("'", "\'") does nothing - as \' string literal is the same as ' in java - what was probably suppose to be here is replace("'", "\\'") - but this would break escaping as it would escape everything including the FreeMarker variables